src/app/[locale]/(pages)/accedi/errore/__test__/errore.test.tsx

```diff
@@ -0,0 +1,117 @@
+import { useSearchParams } from 'next/navigation';
+import { fireEvent, screen } from '@testing-library/react';
+import LoginErrorPage from '../page';
+import { pushMock } from '../../../../../../../jest.setup';
+import { renderWithProviders } from '@/app/[locale]/_utils/test-utils';
+import { ROUTES } from '@/app/[locale]/_utils/routes';
+
+// Mock the useSearchParams function
+jest.mock('next/navigation', () => ({
+  useSearchParams: jest.fn(),
+}));
+
+// Mock the behavior of useSearchParams
+const mockSearchParams = {
+  get: jest.fn(),
+};
+
+(useSearchParams as jest.Mock).mockReturnValue(mockSearchParams);
+
+describe('test suite for Accedi Errore ', () => {
```

Comment:
Here we are mixing English and Italian. Is this wanted? I mean in the test description, not in the expected values.

Reply:
It's a mistake, let's fix it